packages/dapp/src/components/Main.tsx

```diff
@@ -118,10 +134,25 @@ class Main extends React.Component<MainReduxProps & DispatchProp<any> & RouteCom
     }
   };
 
+  public allowTracking = (ev: any): void => {
+    this.answerTracking();
+    cookie.save("allowTracking", true, { path: "/" });
+    addMiddleware(gaMiddleware);
+    this.setState({ cookieSet: true });
+  };
+
+  public denyTracking = (ev: any): void => {
+    this.answerTracking();
+    cookie.save("allowTracking", false, { path: "/" });
```

Comment:
I think we might want the `allowTracking` cookie to use the same `maxAge` value as the `didAnswerTracking` cookie. And maybe set that as a variable or constant in this file.

Reply:
makes sense